common/source/docs/common-esc-telemetry.rst

```diff
@@ -0,0 +1,98 @@
+.. _common-esc-telemetry:
+
+=============
+ESC Telemetry
```

Comment:
I wonder if we're trying to cover all the various protocols that support ESC Telemetry (in the wider sense) or do we just want to cover BLHeli's ESC telemetry feature.  If it's the former then we should expand this to include the CAN protocols.  If it's the latter then we should perhaps change the title to BLHeli ESC Telemetry.

Reply:
I disagree....there are many ESCs that provide telemetry that are _**not**_ BLHeli and that ArduPilot currently does not support (we probably should)...Castle, HobbyWing, etc. I can add CAN and notes on the BiDir that it is _**currently**_ only in BLHeli escs (I have been told recently that its coming on non-BLHeli ESC shortly...but I cant find that note right now)